salt/orchestrate/aws/app_elb.sls

```diff
@@ -10,7 +10,7 @@
         name=env_data.vpc_name).vpcs[0].id
     ).subnets|map(attribute='id')|list %}
 {% set zone_names = salt.boto_route53.describe_hosted_zones|map(attribute='Name')|list %}
-{% set domains = [purpose.domain] %}
+{% set domains = [purpose.domains] %}
```

Comment:
The value of `purpose.domains` is already a list so wrapping it in `[]` will cause it to be a nested list.